src/bin/pg_autoctl/monitor.c

```diff
@@ -1402,12 +1448,12 @@ printCurrentState(void *ctx, PGresult *result)
 
 
 /*
- * monitor_get_state_as_json returns a single string that contains the JSON
- * representation of the current state on the monitor.
+ * monitor_print_state_as_json prints to given stream a single string that
+ * contains the JSON representation of the current state on the monitor.
  */
 bool
-monitor_get_state_as_json(Monitor *monitor, char *formation, int group,
-						  char *json, int size)
+monitor_print_state_as_json(Monitor *monitor, char *formation, int group,
+							FILE *stream)
```

Comment:
can we drop `*stream` and use `stdout` directly here ?

Reply:
Done now, good catch, thanks!